### PR TITLE
Fix more warnings

### PR DESCRIPTION
**Pull request by kain88-de**
Fixes #

Changes made in this Pull Request:
 - fixup instant selector usage

PR Checklist
------------
 - [ ] Tests?
 - [ ] Docs?
 - [ ] CHANGELOG updated?
 - [ ] Issue raised/referenced?

### CODE REVIEW

File: testsuite/MDAnalysisTests/analysis/test_rms.py [hunk 1]
```diff
@@ -358,7 +358,9 @@ def test_rmsf_single_frame(self, universe):
                             err_msg="error: rmsfs should all be zero")
 
     def test_rmsf_old_run(self, universe):
-        rmsfs = rms.RMSF(universe.select_atoms('name CA')).run(start=5, stop=6)
+        with pytest.deprecated_call():
+            rmsfs = rms.RMSF(universe.select_atoms('name CA')).run(start=5,
+                                                                   stop=6)
```
Comment (tylerjereddy) [inline testsuite/MDAnalysisTests/analysis/test_rms.py]: Would it be helpful to add a brief comment indicating why this is deprecated (i.e., how this should be done now)?

**Comment by kain88-de** [comment]
The run method gives an appropriate message like you suggested.

**Comment by tylerjereddy** [comment]
@kain88-de You mean that it issues an appropriate deprecation warning message? I'd have expected as much -- I just meant that a dev looking at the tests may want to know quickly why the call signature is deprecated without executing the code to figure out that run arguments are now deprecated (I didn't know without checking source).

**Comment by tylerjereddy** [comment]
Merged, thanks @kain88-de

**Pull request merged**